src/components/CStreamDetailsStatus/index.tsx

```diff
@@ -0,0 +1,19 @@
+import cn from 'classnames';
+interface CStreamDetailsStatusProps {
+  type: 'Active' | 'Expired' | 'Pending';
+}
+
+const CStreamDetailsStatus = ({ type }: CStreamDetailsStatusProps) => {
+  let statusStyle;
```

Comment:
give status style a default value (pending for example, and then remove the last else if)